projects/cds/src/profiletag/services/profile-tag.injector.spec.ts

```diff
@@ -12,8 +12,8 @@ import {
   WindowRef,
 } from '@spartacus/core';
 import { BehaviorSubject } from 'rxjs';
-import { CdsConfig } from '../../config/cds-config';
-import { ProfileTagWindowObject } from '../model/profile-tag.model';
+import { CdsConfig } from '../../config';
```

Comment:
Please import from the barrel file or directly from the file

Reply:
I guess it defaults to the barrel file if you specify the folder, but I will change it